editor/multi_node_edit.cpp

```diff
@@ -63,9 +68,11 @@ bool MultiNodeEdit::_set_impl(const StringName &p_name, const Variant &p_value,
 		}
 
 		if (p_value.get_type() == Variant::NODE_PATH) {
-			Node *tonode = n->get_node(p_value);
-			NodePath p_path = n->get_path_to(tonode);
-			ur->add_do_property(n, name, p_path);
+			NodePath path;
```

Comment:
I wonder if it's possible to move the above code here.

```c++
Node *node_path_target = nullptr;
if (p_value.get_type() == Variant::NODE_PATH && p_value != NodePath()) {
node_path_target = es->get_node(p_value);
}
```

Reply:
Sure it's possible. I moved that part outside of the loop just because there's no need to keep calling `es->get_node(p_value)` on each iteration. But yeah, I guess performancewise it wouldn't be a problem (unless dealing with enormous number of selected nodes and/or complicated path). So yeah, I can move everything into the loop if that's preferred stylewise.

Reply:
If you wanted to optimize, you would cache the `es->get_node(p_value)` as an object id.

Reply:
I don't understand what you're suggesting / don't see benefit from using object id, can you elaborate?

Reply:
If that code can be outside the loop then it's better outside. Also object id has no benefit, rather the opposite.